Misc/NEWS.d/next/Security/2019-09-25-13-21-09.bpo-38243.1pfz24.rst

```diff
@@ -0,0 +1,2 @@
+Escape the server_title of :class:`DocXMLRPCServer` when rendering the
+document page. (Contributed by Dong-hee Na in :issue:`38243`.)
```

Comment:
I suggest:

Escape the server title of :class:`xmlrpc.server.DocXMLRPCServer` when rendering the
document page as HTML. (Contributed by Dong-hee Na in :issue:`38243`.)